src/routes/resources/+page.svelte

```diff
@@ -4,7 +4,34 @@
   export let data: PageData;
 
   let resources = data.payload.resources;
-  let displayedResources = resources;
+  var beginCount = 0;
+  var lastElem = 20;
```

Comment:
So that it's an even multiple of 3 and 2 (the # of resources per row at different screen sizes)
```suggestion
  var lastElem = 18;
```

Reply:
I did make the changes that you suggested